src/Providers/ReviewsIo/Provider.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace PodPoint\Reviews\Providers\ReviewsIo;
+
+use PodPoint\Reviews\ActionsInterface;
+use PodPoint\Reviews\ProviderInterface;
+
+/**
+ * Class Provider
+ * @package PodPoint\Reviews\Providers\ReviewsIo
+ */
+class Provider implements ProviderInterface
+{
+    /**
+     * @var array
+     */
+    protected $config;
+
+    /**
+     * @var ReviewsCoUkApiClient
+     */
+    protected $apiClient;
+
+    /**
+     * ReviewsProvider constructor.
+     *
+     * @param array $config
+     */
+    public function __construct(array $config)
+    {
+        $this->config = $config;
+
+        $this->apiClient = new ReviewsCoUkApiClient(
+            $config['api_key']
+        );
+    }
+
+    /**
+     * Returns service actions by setting config.
+     *
+     * @return ActionsInterface
+     */
+    public function service(): ActionsInterface
```

Comment:
I think here and in the other provider, as well as their DocBlocks we should rename service to merchant to match the new file name